trulens_eval/trulens_eval/react_components/record_viewer/__init__.py

```diff
@@ -6,7 +6,7 @@
 # the component, and True when we're ready to package and distribute it.
 # (This is, of course, optional - there are innumerable ways to manage your
 # release process.)
-_RELEASE = True
```

Comment:
Could we keep _RELEASE = True in the commits?

Reply:
thanks for catching it - my bad